src/script/test-ceph-backport.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if git $current_pr stats | grep ceph-backport.sh ; then
```

Comment:
Where does `$current_pr` come from?

Reply:
it used gitea, which works similar to github API is what I think, but I am not sure, pinged Loic on #ceph-backports